src/tiled/mapview.cpp

```diff
@@ -233,34 +233,35 @@ void MapView::wheelEvent(QWheelEvent *event)
         setTransformationAnchor(QGraphicsView::AnchorViewCenter);
         return;
     }
-
```

Comment:
Please don't remove the empty line and don't turn the rest of the function into an `else` block. It's not necessary because of the `return` above.